src/ResourceManager/ServiceBus/Commands.ServiceBus/help/New-AzureRmServiceBusQueue.md

```diff
@@ -131,6 +131,22 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -EnableBatchedOperations
+Enable Batched Operations - value that indicates whether server-side batched operations are enabled
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+Accepted values: TRUE, FALSE
```

Comment:
@v-Ajnava make sure to regenerate the help for these two cmdlets after you remove the unnecessary properties on the parameters

Reply:
Done, the extra properties got removed when removed from cmd file.